legend-engine-language-pure-dsl-service/src/main/java/org/finos/legend/engine/language/pure/dsl/service/grammar/to/HelperServiceGrammarComposer.java

```diff
@@ -134,8 +134,9 @@ private static String renderKeyedSingleExecution(KeyedSingleExecutionTest keyedS
     private static String renderTestContainers(List<TestContainer> testContainers, int indentation, PureGrammarComposerContext context)
     {
         StringBuilder builder = new StringBuilder().append(getTabString(indentation)).append("[\n");
-        // TODO: handle parameters
-        builder.append(LazyIterate.collect(testContainers, testContainer -> getTabString(indentation + 1) + "{ " + "[]" + ", " + testContainer._assert.accept(DEPRECATED_PureGrammarComposerCore.Builder.newInstance(context).build()) + " }").makeString(",\n")).append(testContainers.isEmpty() ? "" : "\n");
+        builder.append(LazyIterate.collect(testContainers, testContainer -> getTabString(indentation + 1) +
```

Comment:
Fix the indentation here to make it more readable. Also, since a string builder is being used, we can try to avoid string concatenation

Reply:
fixed